frontend/src/modules/Dashboard/Components/Dashboard.tsx

```diff
@@ -28,6 +28,7 @@ type SortOrder =
   | 'classes-a-z'
   | 'classes-z-a'
   | 'no-check-in-email'
+  | 'no-match-email'
```

Comment:
The wording of this is a little inconsistent with the previous sort order type for check in emails. That was (no) check in email and this is (no) match email, except it's like (no) no match yet email